.github/workflows/publish_artifact_and_create_update_pr.yaml

```
@@ -7,8 +7,17 @@ on:
 
 permissions:
   pull-requests: write
+  packages: write
 
 jobs:
+  deploy_artifact:
+    name: Deploy github package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ./.github/actions/publish-maven_artifacts
```

```suggestion
      - uses: ./.github/actions/publish-maven-artifacts
```

`-` in place of `_`, right?